templates/index.html

```diff
@@ -2,16 +2,31 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+  <div class="alert alert-success" role="alert">
+    <h1 class="alert-heading">To-Do App</h1>
+    <p class="mb-0">Just another to-do app.</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+          {% for todo in todos %}
+            {{todo.id}} - {{todo.title}}
+            <br>
```

Comment:
You should put line 15 inside a list item element (`<li>...</li>`). This will give you line breaks, so you can get rid of the `<br>` and also it's more semantic (would help in various ways including custom styling and screen readers).

If you don't like the bullet point styling, add styling `list-style-type: none;` to the `ul`, or use an `ol` instead of `ul` for numbered items

(li stands for list item, ul stands for unordered list, ol stands for ordered list)